crates/api_models/src/user.rs

```diff
@@ -150,17 +151,24 @@ pub struct UserOrgMerchantCreateRequest {
     pub merchant_name: Secret<String>,
 }
 
-#[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone, ToSchema)]
 pub struct PlatformAccountCreateRequest {
+    #[schema(max_length = 64, value_type = String, example = "organization_abc")]
     pub organization_name: Secret<String>,
 }
 
-#[derive(Debug, serde::Deserialize, serde::Serialize)]
+#[derive(Debug, serde::Deserialize, serde::Serialize, ToSchema)]
 pub struct PlatformAccountCreateResponse {
+    #[schema(value_type = String, max_length = 64, min_length = 1, example = "org_abc")]
     pub org_id: id_type::OrganizationId,
+    #[schema(value_type = String, example = "organization_abc")]
```

Comment:
```suggestion
    #[schema(value_type = Option<String>, example = "organization_abc")]
```

Reply:
@AnuthaDev We do not need to mention here Option<String> because the value_type is actually string, option just indicates it might or might not be present. If present the type will be string and that is how it will be implemented as string in open api spec json file